Sources/CwlPreconditionTesting/CwlCatchBadInstruction.swift

```diff
@@ -171,7 +171,10 @@ public func catchBadInstruction(in block: @escaping () -> Void) -> BadInstructio
 		}
 		defer {
 			// 7. Cleanup the mach port
-			mach_port_destroy(mach_task_self_, context.currentExceptionPort)
+            // When the reference count for the right goes down to 0, it triggers the right to be deallocated
+            mach_port_mod_refs(mach_task_self_, context.currentExceptionPort, MACH_PORT_RIGHT_RECEIVE, -1)
+            // All rights deallocated, can deallocate the name/port
+            mach_port_deallocate(mach_task_self_, context.currentExceptionPort)
```

Comment:
I'm curious about this change as I don't see any warnings or documentation about the deprecation of mach_port_destroy. Can you tell me where this came from?

Reply:
Aah, nevermind... I've found it:

```
@available(iOS, introduced: 2.0, deprecated: 15.0, message: "Inherently unsafe API: instead manage rights with mach_port_destruct(), mach_port_deallocate() or mach_port_mod_refs()")
public func mach_port_destroy(_ task: ipc_space_t, _ name: mach_port_name_t) -> kern_return_t
```

I'll restore this change before creating the tag.